br/pkg/restore/data.go

```diff
@@ -81,15 +94,18 @@ type Recovery struct {
 	totalStores  int
```

Comment:
https://github.com/pingcap/tidb/pull/37406/files#diff-2d7a16b6c7aa66c939ade00f5d5699d0eaa224cb8b0bb69fa34dc854c12e1889R83-R91
this struct is useless

Reply:
RecoveryPlan in previous lines is useless